readme.md

```diff
@@ -269,6 +269,20 @@ The following properties are defined:
 * `endpoint`: Base URL for the API, without e.g. the `index/graphql` suffix.
 * `apiVersion`: One of `otp1`, `otp2` or `entur`.
 
+#### TRIAS
+
+*WIP*
+
+The following properties are defined:
+* `endpoint`: Base URL for the API
+* `testingEndpoint`: Base URL for testing purposes
+* `apiVersion`: `1.X` as [available](https://github.com/VDVde/TRIAS/releases)
+* `content-type`: [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) e.g. `text/xml`
```

Comment:
Is this about the request or response?

Reply:
This is about the request as some endpoints especially mention that this is required, see e.g. [here](https://www.vrn.de/opendata/node/115). I'm not sure about the response. Will check whether there are endpoints that differ in content-type of response and check whether there are clients, which differ in content-type of response and request. If the latter would exist, we would need two properties.

Reply:
Even if the latter doesn't exist, I'd rename it to e.g. `request-content-type`.